bluebrain/repo-bluebrain/packages/nmodl/package.py

```diff
@@ -15,6 +15,7 @@ class Nmodl(CMakePackage):
 
     version("develop", branch="master", submodules=True)
     version("llvm", branch="llvm", submodules=True)
+    version("0.6.0", tag="0.6")
     version("0.5.c", commit="c3b0736")
     version("0.5.b", commit="243453d")
     version("0.5.a", commit="94cba1c")
```

Comment:
Can all these alpha versions go the way of the dodo?

That is the way.

Reply:
This is the way.